doc/notifications.md

```diff
@@ -55,11 +71,19 @@ if ($type === "state_changed") {
     $gateUrl = 'https://demo.gate.thepay.cz/'; // production: 'https://gate.thepay.cz/'
 
     $config = new TheConfig($merchantId, $projectId, $apiPassword, $apiUrl, $gateUrl);
-    $thePay = new TheClient($config);
-
-    $payment = $thePay->getPayment($uid);
+    $signatureService = new SignatureService($config);
```

Comment:
The only thing we need here is getPayment and the condition bellow. Everything else is in README.md and we just could add a reference to specific section in README.md before this example.

I think that the most of developers have some method for making TheClient instance in one place and we need to show here the important part of code they insterested in.

Reply:
I reduced example but let here example of making TheConfig, because in notification and user return is projectId taken from HTTP request.